backends/http.go

```diff
@@ -158,11 +169,11 @@ func (o HTTP) CheckAcl(username, topic, clientid string, acc int32) bool {
 		"acc":      []string{strconv.Itoa(int(acc))},
 	}
 
-	return httpRequest(o.Host, o.AclUri, username, o.WithTLS, o.VerifyPeer, dataMap, o.Port, o.ParamsMode, o.ResponseMode, urlValues)
+	return o.httpRequest(o.Host, o.AclUri, username, o.WithTLS, o.VerifyPeer, dataMap, o.Port, o.ParamsMode, o.ResponseMode, urlValues)
 
 }
 
-func httpRequest(host, uri, username string, withTLS, verifyPeer bool, dataMap map[string]interface{}, port, paramsMode, responseMode string, urlValues map[string][]string) bool {
+func (o HTTP) httpRequest(host, uri, username string, withTLS, verifyPeer bool, dataMap map[string]interface{}, port, paramsMode, responseMode string, urlValues map[string][]string) bool {
```

Comment:
This function no longer needs params that are associated to the HTTP backend as it's now a method of it and it can access them.